frontEnd/tickets-app/src/components/App/App.js

```diff
@@ -0,0 +1,16 @@
+import Header from "../Header/Header";
+import Main from "../Main/Main";
+import Footer from "../Footer/Footer";
+import React, { Fragment } from "react";
+
+const App = () => {
+  return (
+    <Fragment>
```

Comment:
Don't use `Fragment` directly, if you don't have attributes, use  `<> </>` instead.

See:
```
<>
 <Component1 />
 <Component2 />
<>
```
is rather than:
```
<React.Fragment>
  <Component1 />
  <Component2 />
</React.Fragment>
```

Those  2 examples are the same, but the first one is preferable